src/tests/JIT/opt/Devirtualization/AnalyzeCsFile_Github_52270.cs

```diff
@@ -0,0 +1,60 @@
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Text;
+
+public static class Program
+{
+    public static async Task<int> Main()
+    {
+        // Make sure it doesn't assert, see https://github.com/dotnet/runtime/issues/52270
+        for (int i = 0; i < 100; i++)
```

Comment:
This test looks very expensive. How long does it take to run? How long in minopts on Linux arm32?

Reply:
Pretty long, will remove it